streamingpro-mlsql/src/main/java/streaming/dsl/TrainAdaptor.scala

```diff
@@ -46,6 +46,7 @@ class TrainAdaptor(scriptSQLExecListener: ScriptSQLExecListener) extends DslAdap
       ctx.getChild(tokenIndex) match {
         case s: TableNameContext =>
           tableName = evaluate(s.getText)
+          options += ("viewName" -> evaluate(s.getText))
```

Comment:
We should not add any magic "name" here.

Reply:
If you wanna get the viewName, use the `tempTable` in the last of train function.